rudder-core/src/main/scala/com/normation/rudder/services/queries/DynGroupUpdaterService.scala

```diff
@@ -104,11 +104,7 @@ class DynGroupUpdaterServiceImpl(
       newMemberIdsSet = newMembers.map( _.id).toSet
       savedGroup <- {
                       val newGroup = group.copy(serverList = newMemberIdsSet)
-                      if(group.isSystem) {
-                        woNodeGroupRepository.updateSystemGroup(newGroup, modId, actor, reason)
-                      } else {
-                        woNodeGroupRepository.update(newGroup, modId, actor, reason)
-                      }
+                      woNodeGroupRepository.updateDynGroupNodes(newGroup, modId, actor, reason)
```

Comment:
if the update fail, it will interupt all the dyngroup update, isn't it ?
shouldn't we simply restart the update in this case ?